cloud/aws/deploytf/deploy.go

```diff
@@ -103,6 +103,18 @@ func (a *NitricAwsTerraformProvider) RequiredProviders() map[string]interface{}
 	return map[string]interface{}{}
 }
 
+func (a *NitricAwsTerraformProvider) GetGlobalTags() *map[string]*string {
+	commonTags := a.CommonStackDetails.Tags
+
+	// merge resourceTags and common tags
```

Comment:
Or remove the comment entirely

```suggestion
	// convert common tags to Terraform variable typing
```